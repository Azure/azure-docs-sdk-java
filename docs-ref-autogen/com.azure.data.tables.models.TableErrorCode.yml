### YamlMime:JavaType
uid: "com.azure.data.tables.models.TableErrorCode"
fullName: "com.azure.data.tables.models.TableErrorCode"
name: "TableErrorCode"
nameWithType: "TableErrorCode"
summary: "Defines values for error codes returned from the Tables service. For detailed information about what each error code means and their accompanying status codes, visit the following page: [Table service error codes][]\n\n\n[Table service error codes]: https://docs.microsoft.com/rest/api/storageservices/table-service-error-codes"
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class TableErrorCode extends ExpandableStringEnum<TableErrorCode>"
constructors:
- "com.azure.data.tables.models.TableErrorCode.TableErrorCode()"
fields:
- "com.azure.data.tables.models.TableErrorCode.ACCOUNT_IOPS_LIMIT_EXCEEDED"
- "com.azure.data.tables.models.TableErrorCode.ATOM_FORMAT_NOT_SUPPORTED"
- "com.azure.data.tables.models.TableErrorCode.AUTHORIZATION_PERMISSION_MISMATCH"
- "com.azure.data.tables.models.TableErrorCode.AUTHORIZATION_RESOURCE_TYPE_MISMATCH"
- "com.azure.data.tables.models.TableErrorCode.CANNOT_CREATE_TABLE_WITH_IOPS_GREATER_THAN_MAX_ALLOWED_PER_TABLE"
- "com.azure.data.tables.models.TableErrorCode.COMMANDS_IN_BATCH_ACT_ON_DIFFERENT_PARTITIONS"
- "com.azure.data.tables.models.TableErrorCode.CONTENT_LENGTH_EXCEEDED"
- "com.azure.data.tables.models.TableErrorCode.DUPLICATE_KEY_PROPERTY_SPECIFIED"
- "com.azure.data.tables.models.TableErrorCode.DUPLICATE_PROPERTIES_SPECIFIED"
- "com.azure.data.tables.models.TableErrorCode.ENTITY_ALREADY_EXISTS"
- "com.azure.data.tables.models.TableErrorCode.ENTITY_NOT_FOUND"
- "com.azure.data.tables.models.TableErrorCode.ENTITY_TOO_LARGE"
- "com.azure.data.tables.models.TableErrorCode.FORBIDDEN"
- "com.azure.data.tables.models.TableErrorCode.INVALID_DUPLICATE_ROW"
- "com.azure.data.tables.models.TableErrorCode.INVALID_INPUT"
- "com.azure.data.tables.models.TableErrorCode.INVALID_VALUE_TYPE"
- "com.azure.data.tables.models.TableErrorCode.JSON_FORMAT_NOT_SUPPORTED"
- "com.azure.data.tables.models.TableErrorCode.JSON_VERBOSE_FORMAT_NOT_SUPPORTED"
- "com.azure.data.tables.models.TableErrorCode.KEY_VALUE_TOO_LARGE"
- "com.azure.data.tables.models.TableErrorCode.MEDIA_TYPE_NOT_SUPPORTED"
- "com.azure.data.tables.models.TableErrorCode.METHOD_NOT_ALLOWED"
- "com.azure.data.tables.models.TableErrorCode.NOT_IMPLEMENTED"
- "com.azure.data.tables.models.TableErrorCode.OPERATION_TIMED_OUT"
- "com.azure.data.tables.models.TableErrorCode.OPERATOR_INVALID"
- "com.azure.data.tables.models.TableErrorCode.OUT_OF_RANGE_INPUT"
- "com.azure.data.tables.models.TableErrorCode.PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED"
- "com.azure.data.tables.models.TableErrorCode.PARTITION_KEY_NOT_SPECIFIED"
- "com.azure.data.tables.models.TableErrorCode.PARTITION_KEY_PROPERTY_CANNOT_BE_UPDATED"
- "com.azure.data.tables.models.TableErrorCode.PARTITION_KEY_SPECIFIED_MORE_THAN_ONCE"
- "com.azure.data.tables.models.TableErrorCode.PER_TABLE_IOPS_DECREMENT_LIMIT_REACHED"
- "com.azure.data.tables.models.TableErrorCode.PER_TABLE_IOPS_INCREMENT_LIMIT_REACHED"
- "com.azure.data.tables.models.TableErrorCode.PRIMARY_KEY_PROPERTY_IS_INVALID_TYPE"
- "com.azure.data.tables.models.TableErrorCode.PROPERTIES_NEED_VALUE"
- "com.azure.data.tables.models.TableErrorCode.PROPERTY_NAME_INVALID"
- "com.azure.data.tables.models.TableErrorCode.PROPERTY_NAME_TOO_LONG"
- "com.azure.data.tables.models.TableErrorCode.PROPERTY_VALUE_TOO_LARGE"
- "com.azure.data.tables.models.TableErrorCode.RESOURCE_NOT_FOUND"
- "com.azure.data.tables.models.TableErrorCode.SETTING_IOPS_FOR_A_TABLE_IN_PROVISIONING_NOT_ALLOWED"
- "com.azure.data.tables.models.TableErrorCode.TABLE_ALREADY_EXISTS"
- "com.azure.data.tables.models.TableErrorCode.TABLE_BEING_DELETED"
- "com.azure.data.tables.models.TableErrorCode.TABLE_HAS_NO_PROPERTIES"
- "com.azure.data.tables.models.TableErrorCode.TABLE_HAS_NO_SUCH_PROPERTY"
- "com.azure.data.tables.models.TableErrorCode.TABLE_NOT_FOUND"
- "com.azure.data.tables.models.TableErrorCode.TOO_MANY_PROPERTIES"
- "com.azure.data.tables.models.TableErrorCode.UPDATE_CONDITION_NOT_SATISFIED"
- "com.azure.data.tables.models.TableErrorCode.X_METHOD_INCORRECT_COUNT"
- "com.azure.data.tables.models.TableErrorCode.X_METHOD_INCORRECT_VALUE"
- "com.azure.data.tables.models.TableErrorCode.X_METHOD_NOT_USING_POST"
methods:
- "com.azure.data.tables.models.TableErrorCode.fromString(java.lang.String)"
type: "class"
metadata: {}
package: "com.azure.data.tables.models"
artifact: com.azure:azure-data-tables:12.2.1
