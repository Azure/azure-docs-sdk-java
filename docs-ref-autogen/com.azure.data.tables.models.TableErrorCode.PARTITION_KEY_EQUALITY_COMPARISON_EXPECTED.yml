### YamlMime:JavaMember
uid: "com.azure.data.tables.models.TableErrorCode.PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED*"
fullName: "com.azure.data.tables.models.TableErrorCode.PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED"
name: "PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED"
nameWithType: "TableErrorCode.PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED"
members:
- uid: "com.azure.data.tables.models.TableErrorCode.PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED"
  fullName: "com.azure.data.tables.models.TableErrorCode.PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED"
  name: "PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED"
  nameWithType: "TableErrorCode.PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED"
  summary: "Static value `PartitionKeyEqualityComparisonExpected`."
  syntax: "public static final TableErrorCode PARTITION_KEY_EQUALITY_COMPARISON_EXPECTED"
type: "field"
metadata: {}
package: "com.azure.data.tables.models"
artifact: com.azure:azure-data-tables:12.2.1
